### PR TITLE
rbd-mirror: additional thrasher testing

### PR DESCRIPTION
**Pull request by dillaman**

### CODE REVIEW

**Comment by dillaman** [comment]
http://pulpito.ceph.com/jdillaman-2018-04-27_08:35:15-rbd-wip-18753-1-distro-basic-smithi/

File: qa/workunits/rbd/rbd_mirror_helpers.sh [hunk 1]
```diff
@@ -858,6 +877,23 @@ compare_images()
...
+    local image=$2
+
+    local rmt_export=${TEMPDIR}/${CLUSTER2}-${pool}-${image}.export
+    local loc_export=${TEMPDIR}/${CLUSTER1}-${pool}-${image}.export
+
+    for snap_name in $(rbd --cluster ${CLUSTER1} -p ${pool} snap list ${image}); do
+        rm -f ${rmt_export} ${loc_export}
+        rbd --cluster ${CLUSTER2} -p ${pool} export ${image}@${snap_name} ${rmt_export}
+        rbd --cluster ${CLUSTER1} -p ${pool} export ${image}@${snap_name} ${loc_export}
+        cmp ${rmt_export} ${loc_export}
```
Comment (trociny) [inline qa/workunits/rbd/rbd_mirror_helpers.sh]: @dillaman You could do something like this:
```
rmt_md5sum=`rbd ... export image - | md5sum`
```
And just compare check sums. It looks like less code and probably a little faster.

Comment (dillaman) [reply qa/workunits/rbd/rbd_mirror_helpers.sh]: I was keeping it consistent w/ https://github.com/ceph/ceph/pull/21697/files/1d5b146cfa6c72f02860f15d1ec330ff97ce0848..de4587012d2d0d35f9d201550d58ffc2faeff27b#diff-1d6828c5e08d9d9d1f6dea4aa3430afaR876. Plus ```cmp``` will at least say the offset of the first delta byte.

**Comment by dillaman** [comment]
@trociny It's for mimic

**Pull request merged**